authentication/views.py

```diff
@@ -37,25 +36,30 @@ def _keycloak_logout_url(self, user):
         qs = urlencode(
             {
                 "id_token_hint": id_token,
-                "post_logout_redirect_uri": settings.LOGOUT_REDIRECT_URL,
+                "post_logout_redirect_uri": self.request.build_absolute_uri(
+                    settings.LOGOUT_REDIRECT_URL
+                ),
             }
         )
-        return f"{settings.KEYCLOAK_BASE_URL}/realms/{settings.KEYCLOAK_REALM_NAME}/protocol/openid-connect/logout?{qs}"  # noqa: E501
+
+        return (
+            f"{settings.KEYCLOAK_BASE_URL}/realms/"
+            f"{settings.KEYCLOAK_REALM_NAME}/protocol/openid-connect/logout"
+            f"?{qs}"
+        )
 
     def get(
         self,
         request,
         *args,  # noqa: ARG002
         **kwargs,  # noqa: ARG002
-    ):  # pylint:disable=unused-argument
```

Comment:
I'm not fully understanding this change. if we absolutely need to do this ahead of django 5 could we instead define a separate post() method that does the same thing?

also worth noting - on the frontend we appear to be logging out via GET on /logout (admin has its own logout flow that is done via post)

Reply:
That was an accidental commit of me trying to make POST work, I remove the changes.